src/content/structured/get-started/angular.mdx

```diff
@@ -7,7 +7,7 @@ date: "2022-11-15"
 
 title: "Angular"
 
-subTitle: "To use the components follow the relevant framework instructions."
+subTitle: "How to use the components in an Angular-based framework."
```

Comment:
I think the wording for this (and Svelte, and Vue) is a little inaccurate. People are making Apps that use a Framework - so i'd suggest the sentence be 
"How to use the components in an Angular-based application"